Cargo.toml

```diff
@@ -1,107 +1,10 @@
-[package]
-name = "ipa"
-version = "0.1.0"
-rust-version = "1.64.0"
-edition = "2021"
+[workspace]
 
-[features]
-# by default remove all TRACE, DEBUG spans from release builds
-default = ["web-app", "in-memory-infra", "tracing/max_level_trace", "tracing/release_max_level_info"]
-cli = ["comfy-table", "clap"]
-enable-serde = ["serde", "serde_json"]
-protocol-perf-testing = ["no-prss"]
-no-prss = []
-disable-metrics = []
-# TODO move web-app to a separate crate. It adds a lot of build time to people who mostly write protocols
-# TODO Consider moving out benches as well
-web-app = [
-    "axum", "axum-server", "base64", "clap", "comfy-table", "enable-serde", "hyper", "hyper-tls",
-    "rcgen", "rustls-pemfile", "sec1", "time", "tokio-rustls", "toml", "tower", "tower-http",
+members = [
+    "ipa",
+    "ipa-macros",
+    "helper",
```

Comment:
I kinda like having it, but curious what @martinthomson thinks. Macros have to be in a separate crate, so that alone justifies the workspace creation, isolating helper service may not be strictly required for now.